tools/validators/instance_validator/tests/fake_instances/GOOD/entity_export_operation.yaml

```diff
@@ -20,5 +20,5 @@ CONFIG_METADATA:
 SDC_EXT-19-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-19
-  cloud_device_id: "baz"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.